cmd/auth-handler.go

```diff
@@ -114,12 +116,14 @@ func checkAdminRequestAuthType(r *http.Request, region string) APIErrorCode {
 		s3Err = isReqAuthenticated(r, region)
 	}
 	if s3Err != ErrNone {
-		errorIf(errors.New(getAPIError(s3Err).Description), "%s", dumpRequest(r))
+		reqInfo := (&logger.ReqInfo{}).AppendTags("requestDump", dumpRequest(r))
```

Comment:
we can perhaps do a better job at this time rather than calling it `requestDump`  we can call it `requestHeaders` its more of collection of headers than a `dump` :-)

Reply:
done